docs/components/docs/GitHubExamplesCTA.tsx

```diff
@@ -25,7 +25,7 @@ export function GitHubExamplesCTA() {
         target="_blank"
         rel="noopener noreferrer"
       >
-        GitHub Repo <ArrowR />
+        Keystone GitHub Repo <ArrowR />
```

Comment:
Sentence case? (this is in part me learning our content guidelines)

Reply:
Fixed in #6528